### PR TITLE
Test for JENKINS-32132 with rule to mock GH

### PR DESCRIPTION
**Pull request by lanwen**
[Review on Reviewable](https://reviewable.io/reviews/jenkinsci/github-plugin/104)

### CODE REVIEW

File: src/test/resources/com/cloudbees/jenkins/GitHubSetCommitStatusBuilderTest/shouldLoadNullStatusMessage/config.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            <filterQueue>false</filterQueue>
+            <properties class="hudson.model.View$PropertyList"/>
+        </hudson.model.AllView>
+    </views>
+    <primaryView>All</primaryView>
+    <slaveAgentPort>0</slaveAgentPort>
+    <label></label>
+    <nodeProperties/>
+    <globalNodeProperties/>
+</hudson>
```
Comment (KostyaSha) [inline src/test/resources/com/cloudbees/jenkins/GitHubSetCommitStatusBuilderTest/shouldLoadNullStatusMessage/config.xml]: NL >_<

File: src/test/resources/log4j.properties [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Root logger option
+log4j.rootLogger=INFO, stdout
+
+# Direct log messages to stdout
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
```
Comment (KostyaSha) [inline src/test/resources/log4j.properties]: Maybe better exclude log4j adapters and add logback?

**Comment by KostyaSha** [comment]
:+1:

File: src/test/resources/log4j.properties (hunk 2 above)
Comment (lanwen) [inline src/test/resources/log4j.properties]: log4j in the core, so no additional actions required for log4j - only configure appenders

**Pull request merged**